spec/factories/case_log.rb

```diff
@@ -152,6 +152,128 @@
       chcharge { 7 }
       letting_in_sheltered_accomodation { "No" }
       la_known { "Yes" }
+      declaration { "Yes" }
+    end
+    trait :completed_without_declaration do
```

Comment:
Nested factory/traits might be better here to avoid duplicating all the data just to change one field? (or even just flipping the field in your test?)

Reply:
That's very fair, was struggling to do that yesterday cause the validation kept on going off. Clearly was missing something cause it seems okay now. Thank you thank you!